CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+# Contributing to the APM Agent
+
+The APM Agent is open source and we love to receive contributions from our community — you!
+
+There are many ways to contribute, from writing tutorials or blog posts, improving the documentation, submitting bug reports and feature requests or writing code.
+
+You can get in touch with us through [Discuss](https://discuss.elastic.co/c/apm), feedback and ideas are always welcome.
+
+## Code contributions
+
+If you have a bugfix or new feature that you would like to contribute, please find or open an issue about it first.
+Talk about what you would like to do.
+It may be that somebody is already working on it, or that there are particular issues that you should know about before
+implementing the change.
+
+You will have to fork the `apm-agent-nodejs` repo, please follow the instructions in the [readme](README.md).
+
+### Submitting your changes
+
+Generally, we require that you test any code you are adding or modifying.
+Once your changes are ready to submit for review:
+
+1. Sign the Contributor License Agreement
+
+    Please make sure you have signed our [Contributor License Agreement](https://www.elastic.co/contributor-agreement/).
+    We are not asking you to assign copyright to us, but to give us the right to distribute your code without restriction. We ask this of all contributors in order to assure our users of the origin and continuing existence of the code.
+    You only need to sign the CLA once.
+
+2. Test your changes
+
+    Run the test suite to make sure that nothing is broken. See [testing](TESTING.md) for details.
```

Comment:
We don't have a `TESTING.md` file either, but just a section in our `README.md`. Right now it's split into two sections:

- https://github.com/elastic/apm-agent-nodejs#testing
- https://github.com/elastic/apm-agent-nodejs#using-docker-for-testing

But it would probably make sense to refactor that part of the README. That way we can also include the information needed for the comment I made above

Reply:
I think it would make sense to move all development/testing related documentation to this file and then instead link to that from the README. Could I get you to move the two sections in below this section? Just make the "Testing" section a level 3 (`###`) and make the other one a level 4 (`####`).

Then add a link at the end of the [Development section](https://github.com/elastic/apm-agent-nodejs#development-notes) in the README to this file.

Reply:
Would it make sense to also have the [test commands](https://github.com/elastic/apm-agent-nodejs/blob/master/README.md#testing-documentation) included here as well?

Reply:
Oh sorry, didn't see this comment before adding my review. Yes, I think it would make sense 😃